windows/src/desktop/setup/UfrmRunDesktop.pas

```diff
@@ -1033,16 +1034,25 @@ procedure TfrmRunDesktop.GetDefaultSettings; // I2651
 begin
   FStartWithWindows := True; // I2607
   FCheckForUpdates := True;  // I2609
+  FAutomaticallyReportUsage := True;
```

Comment:
You have the default `True` here but seems to be `False` in utilkeymanoption.pas?

Reply:
see this commit message https://github.com/keymanapp/keyman/pull/13215/commits/ba22be570e3fab777bee78a797889f0eaeb3a8d5

Reply:
I'm still not really 100% clear (and I think I wrote the code), but I don't understand why this would differ from the koAutomaticallyReportErrors flag?

Reply:
Well if koAutomaticallyReportErrors was an option on a fresh install offered to the user then it would have the same issue. Infact your question led me to realise `FCheckForUpdates` is probably also being set to true even when the user unchecks it on initial install. Testing confirmed it has the same problem. As would `FStartWithWindows` but is set to `False` in utilkeymanoptions.pas.     

An alternate approach may be to set these options here to False here. [if DoAutomaticallyReportUsage then kmcom.Options['koAutomaticallyReportUsage'].Value := True;](https://github.com/keymanapp/keyman/blob/263f918ea21ddaac1bc007a8ab8a672d21e74162/windows/src/desktop/kmshell/main/UImportOlderVersionSettings.pas#L149-L150)

if the flag wasn't set in the command line argument.

Reply:
> Testing confirmed it has the same problem

do we need yet another issue?

Reply:
I will fix this using the alternate approach I mentioned above and will fix the other two options at the same time.